src/db/connectors.js

```diff
@@ -1,7 +1,7 @@
 const casual = require('casual');
 const _ = require('lodash');
 
-const batchUpdate = require('../database_update/batch');
+const batchUpdate = require('../services/db_update');
 
```

Comment:
could we give `batchUpdate` a more descriptive/specific name of what it's doing? It's very vague/general right now